dtfs-central-api/src/helpers/get-previous-report-period.test.ts

```diff
@@ -11,7 +11,7 @@ describe('getPreviousReportPeriod', () => {
     jest.resetAllMocks();
   });
 
-  const utilisationReport = UtilisationReportEntityMockBuilder.forStatus('PENDING_RECONCILIATION').build();
+  let utilisationReport = UtilisationReportEntityMockBuilder.forStatus('PENDING_RECONCILIATION').build();
```

Comment:
We shouldn't use let here and then redeclare in the tests as then the value of the report will be different within each test depending on the order the tests are run.